content/blog/2024-12-21-how-do-i-...-in-dt-5.0/index.md

```diff
@@ -0,0 +1,508 @@
+
+To straighten an image, right-click on your image and drag along a
+line which _should_ be horizontal or vertical, then release the mouse
+button.  darktable will then rotate your image to make the line along
+which you dragged exactly horizontal or vertical.  You can also use
+the _rotation_ slider in the
+[_rotate and perspective_](https://darktable.org/usermanual/en/module-reference/processing-modules/rotate-perspective/),
+which is shown in the
+[_Quick Access Panel_](https://darktable.org/usermanual/en/darkroom/organization/quick-access-panel/)
+by default.
+
+To rotate the image by multiples of 90 degrees, use the
+[_orientation_](https://darktable.org/usermanual/en/module-reference/processing-modules/orientation/)
+module.
+
+</details>
+
+<details><summary><b>adjust contrast/brightness/saturation</b></summary>
+
+To adjust contrast, you can use
+
+- the contrast slider in either
+  [_filmic rgb_](https://darktable.org/usermanual/en/module-reference/processing-modules/filmic-rgb/)'s
+  _look_ tab or [_sigmoid_](https://darktable.org/usermanual/en/module-reference/processing-modules/sigmoid/)
+  (on the [_Quick Access Panel_](https://darktable.org/usermanual/en/darkroom/organization/quick-access-panel/)
+  by default when workflow is set to either of the "scene-referred" options in preferences).
+
+- the contrast slider in [_color balance rgb_](https://darktable.org/usermanual/en/module-reference/processing-modules/color-balance-rgb/),
+  adjusting the _contrast gray fulcrum_ slider on the _masks_ tab to control the center point of the contrast.
+
+- the [_contrast equalizer_](https://darktable.org/usermanual/en/module-reference/processing-modules/contrast-equalizer/)
+  module for more localized contrast.
+
+- the [_local contrast_](https://darktable.org/usermanual/en/module-reference/processing-modules/local-contrast/)
+  module for very localized contrast enhancement, also known as
+  "clarity" (on the [_Quick Access Panel_](https://darktable.org/usermanual/en/darkroom/organization/quick-access-panel/)
+  by default).
+
+To adjust brightness, you can use
+
+- the [_exposure_](https://darktable.org/usermanual/en/module-reference/processing-modules/exposure/) module for linear adjustment
+
+- the _brilliance grading_ sliders in
+  [_color balance rgb_](https://darktable.org/usermanual/en/module-reference/processing-modules/color-balance-rgb/)
+  for perceptual brightness adjustments (the "global brilliance" slider is
+  on the Quick Access Panel by default).
+
+To adjust saturation, you can use
+
+- the _global vibrance_, _chroma grading_, or _saturation grading_ sliders in
+  [_color balance rgb_](https://darktable.org/usermanual/en/module-reference/processing-modules/color-balance-rgb/)
+  (on the Quick Access Panel by default).
+
+- the [_velvia_](https://darktable.org/usermanual/en/module-reference/processing-modules/velvia/) module
+
+</details>
+
+<details><summary><b>adjust shadows/highlights</b></summary>
+
```

Comment:
seems like this should be rolled into the last section

Reply:
Not really, as these are limited to specific tonal ranges, and shadows/highlights is a specific term people would be looking for.